### ISSUE TITLE
Link documentation on addressing performance regressions from bisect bug comments

### ISSUE THREAD

**Issue by anniesullie** [catapult-project/catapult#3658]
We've added documentation about addressing performance regressions:

https://chromium.googlesource.com/chromium/src/+/master/docs/speed/addressing_performance_regressions.md

We should link this when the bisect bot finds a regression. Maybe with a bit.ly link so it's easy to remember?

@simonhatch @benshayden @eakuefner @benhenry

**Comment by simonhatch** [comment]
Where should this go:

=== BISECT JOB RESULTS ===
<stuff>

Revision             Result                  N
chromium@463506      1900.62 +- 181.567      21      good
chromium@464066      1873.34 +- 293.879      21      bad

To Run This Test
  src/tools/perf/run_benchmark -v --browser=release --output-format=chartjson --upload-results --pageset-repeat=1 --also-run-disabled-tests v8.runtimestats.browsing_desktop_classic

Debug Info
  https://chromeperf.appspot.com/buildbucket_job_status/8977279909478202736

Is this bisect wrong?
  https://chromeperf.appspot.com/bad_bisect?try_job_id=5320125470736384

Addressing Performance Regressions
  https://link_to_this_doc_via_bitly?

| O O | Visit http://www.chromium.org/developers/speed-infra/perf-bug-faq
|  X  | for more information addressing perf regression bugs. For feedback,
| / \ | file a bug with component Speed>Bisection.  Thank you!

**Comment by anniesullie** [comment]
I think we should actually change it around a bit and remove cruft:

```
=== BISECT JOB RESULTS ===
Revision Result N
chromium@463506 1900.62 +- 181.567 21 good
chromium@464066 1873.34 +- 293.879 21 bad

To Run This Test
src/tools/perf/run_benchmark -v --browser=release --output-format=chartjson --upload-results --pageset-repeat=1 --also-run-disabled-tests v8.runtimestats.browsing_desktop_classic

More information on addressing performance regressions:
https://link_to_this_doc_via_bitly

Debug information about this bisect:
https://chromeperf.appspot.com/buildbucket_job_status/8977279909478202736
For feedback, file a bug with component Speed>Bisection
```

**Comment by benhenry** [comment]
I think we want a g.co short link, not anything with a TLD in Libya.

**Comment by anniesullie** [comment]
We can apply for a g.co short link, but bit.ly is pretty commonly used (i.e. bit.ly/blinkon6)

**Comment by benhenry** [comment]
I don't think we should be using Libyan TLDs. I remember hearing this is a no-no at a TGIF of the past. What about goo.gl?

**Comment by anniesullie** [comment]
Same application process as g.co.
